src/css/custom.scss

```diff
@@ -100,7 +100,7 @@ body,
     }
 }
 
-.main-wrapper [role='complementary'] {
+.main-wrapper [class^='docSidebarContainer'] {
```

Comment:
This was the cause of breaking styling with the sidebar. Docusaurus is spitting our slightly different html & attributes (swapped `<div role="complementary">` for `<aside>`)